google-cloud-spanner-hibernate-testing/src/main/java/com/google/cloud/spanner/hibernate/SpannerColumnTypeTest.java

```diff
@@ -0,0 +1,5 @@
+package com.google.cloud.spanner.hibernate;
+
+public class SpannerColumnTypeTest {
+
```

Comment:
Were you planning to add something here?

Reply:
My b, forgot I didn't actually add the test. Integration test added [here.](https://github.com/GoogleCloudPlatform/google-cloud-spanner-hibernate/pull/232/files#diff-82a61b6b278d60a08229a48a612fb37365a5fb97bb87bbd56e1c46c3cbbfb137)